client-js/src/utils.ts

```diff
@@ -0,0 +1,3 @@
+import { CLValueParsers } from "casper-js-sdk";
+
+const isObject = (item: unknown) => typeof item === "object";
```

Comment:
Seems to be a bit of overkill to have this as a util. Weere do you use it ?

Reply:
Right - I missed it - removed.